src/index.js

```diff
@@ -145,4 +145,13 @@ export default class BreakLine {
       title: 'Break Line',
     };
   }
+
+  /**
+   * Returns true to notify the core that read-only mode is supported
+   *
+   * @return {boolean}
+   */
+  static get isReadOnlySupported() {
```

Comment:
@MrJuanGaviriaK Could you please add the corresponding tests cases? 

Thank you!

Reply:
@juanhurtado10 

I agree, I'm going to work on it!!!

Thanks